packages/contracts-bedrock/foundry.toml

```diff
@@ -20,7 +20,7 @@ remappings = [
   'ds-test/=lib/forge-std/lib/ds-test/src',
   'safe-contracts/=lib/safe-contracts/contracts',
   'kontrol-cheatcodes/=lib/kontrol-cheatcodes/src',
-  'solady/=lib/solady/src',
+  '@solady/=lib/solady/src',
```

Comment:
We should make sure lib-keccak and the monorepo are both pinned to the same solady version for consistency + in case forge picks up the wrong one

Reply:
Going to follow up on this, getting this merged now to unblock builds